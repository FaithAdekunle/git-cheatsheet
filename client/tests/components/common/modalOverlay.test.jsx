import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ModalOverlay from '../../../src/app/components/common/modalOverlay';

describe('<ModalOverlay />', () => {
  test('should match snapshot', () => {
    const wrapper = shallow(<ModalOverlay />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
