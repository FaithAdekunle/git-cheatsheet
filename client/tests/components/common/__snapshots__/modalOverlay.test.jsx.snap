// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ModalOverlay /> should match snapshot 1`] = `
<div
  className="modal-overlay"
/>
`;
